src/main/java/com/exasol/adapter/installer/dialect/Db2DialectProfile.java

```diff
@@ -0,0 +1,57 @@
+package com.exasol.adapter.installer.dialect;
+
+import com.exasol.adapter.installer.UserInput;
+
+/**
+ * A DB2 dialect profile.
+ */
+public class Db2DialectProfile extends AbstractVirtualSchemaProfile {
+    /**
+     * Instantiate a new {@link Db2DialectProfile}.
+     *
+     * @param userInput user input
+     */
+    public Db2DialectProfile(final UserInput userInput) {
+        super(userInput);
+    }
+
+    @Override
+    protected String getDriverMain() {
+        return "com.ibm.db2.jcc.DB2Driver";
+    }
+
+    @Override
+    protected String getDriverPrefix() {
+        return "jdbc:db2:";
+    }
+
+    @Override
+    protected boolean isSecurityManagerEnabled() {
+        return false;
+    }
+
+    @Override
+    protected String getDefaultPort() {
+        return "50000";
+    }
+
+    @Override
+    protected String getDefaultDriverName() {
+        return "jcc.jar";
+    }
+
+    @Override
+    protected boolean isConfigRequired() {
+        return true;
+    }
+
+    @Override
+    public String getConnectionString() {
```

Comment:
this one looks like the only method that is a bit more tricky to do using configuration. But it's equal in the existing versions right? So maybe yoiu could just move it to the abstract base?

If you replace this by configuration you could maybe define this by with placeholders.